lesson18/index.js

```diff
@@ -1,6 +1,10 @@
-const url = "https://myjson.dit.upm.es/api/bins/8b9d";
+// const url = "https://myjson.dit.upm.es/api/bins/8b9d";
+// const url = "test.json";
+const url = "https://mocki.io/v1/0295f412-b9f3-4247-8a23-ff5a5f88248d";
```

Comment:
MUST
The paths for codesandbox and img are different, please make them the same.
Please check.

Reply:
[3fb8e15](https://github.com/hippopo-moo/js-handson-work/pull/20/commits/3fb8e1569c01cb8fd42520751da79d2753a69710)

Sorry for the lack of confirmation.
The API endpoints have been corrected.